hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/MapNearCacheManager.java

```diff
@@ -134,13 +134,14 @@ public void shutdown() {
      */
     @Override
     public boolean destroyNearCache(String mapName) {
-        if (!super.destroyNearCache(mapName)) {
-            return false;
-        }
-
         String uuid = nodeEngine.getLocalMember().getUuid();
-        invalidator.destroy(mapName, uuid);
-        return true;
+        invalidator.invalidateAllKeys(mapName, uuid);
```

Comment:
Could `invalidateAllKeys` be invoked when destroying a map proxy without any near cache configured?